### PR TITLE
error boundary test spec

### PR DESCRIPTION
**Pull request by nicoglennon**
Add e2e testing for error boundary screen in case of JS error.

### CODE REVIEW

File: e2e/errorBoundary.spec.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    await Helpers.waitAndTap('developer-section');
+    await Helpers.checkIfVisible('developer-settings-modal');
+  });
+
+  it('Should crash the app and show the error boundary section', async () => {
+    await Helpers.waitAndTap('crash-app-section');
+    await Helpers.checkIfVisible('error-boundary-container', 5000);
+  });
+
+  it('Should restart app in Wallet Screen after tapping Restart Rainbow', async () => {
```
Comment (mrbaguvix) [inline e2e/errorBoundary.spec.js:40]:
```suggestion
  it('Should restart app in Wallet Screen after tapping "Restart Rainbow"', async () => {
```

File: e2e/errorBoundary.spec.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    await Helpers.enableSynchronization();
+  });
+
+  it('Should navigate to Settings Modal after tapping Settings Button', async () => {
+    await Helpers.swipe('wallet-screen', 'right');
+    await Helpers.waitAndTap('settings-button');
+    await Helpers.checkIfVisible('settings-modal');
+  });
+
+  it('Should navigate to Developer Settings after tapping Developer Section', async () => {
```
Comment (mrbaguvix) [inline e2e/errorBoundary.spec.js:30]:
```suggestion
  it('Should navigate to "Developer Settings" after tapping "Developer Section"', async () => {
```

File: e2e/errorBoundary.spec.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      // Set the pin
+      await Helpers.authenticatePin('1234');
+      // Confirm it
+      await Helpers.authenticatePin('1234');
+    }
+    await Helpers.checkIfVisible('wallet-screen', 40000);
+    await Helpers.enableSynchronization();
+  });
+
+  it('Should navigate to Settings Modal after tapping Settings Button', async () => {
```
Comment (mrbaguvix) [inline e2e/errorBoundary.spec.js:24]:
```suggestion
  it('Should navigate to Settings Modal after tapping "Settings" Button', async () => {
```

**Review by mrbaguvix — COMMENTED** [review]
I love the tests! The only thing I'm not sure about is we are having some unrelated tests/assertions for  the "Error boundary flow"

**Review by osdnk — APPROVED** [review]
I'm down for those changes. Can you resolve conflicts so we can merge?

**Comment by osdnk** [comment]
They're not passing tho

**Comment by jinchung** [comment]
Closing for now
